### ISSUE TITLE
Error: while running jar file

### ISSUE THREAD

**Issue by yamatokataoka** [petkivim/x-road-test-client#3]
Hello @petkivim,

I just ran jar file with `java -jar x-road-test-client-0.0.6.jar`.
However, I got an error below.
`Error: Could not find or load main class com.pkrete.xrd4j.tools.test_client.Main`

Does the misspelling effect on the jar file? 

Regards,
Yamato

**Comment by petkivim** [comment]
Hi @yamakataoka 

Yes, the error is caused by a typo in the `mainClass` configuration in `pom.xml`. The error is fixed in the current `0.0.7-SNAPSHOT` version.

If you want the use the `0.0.6` release version, please use the command below:

```
java -cp x-road-test-client-0.0.6.jar com.pkrete.xrd4j.tools.testclient.Main
```

Regards,
Petteri

**Comment by yamato-me** [comment]
Thank you for your suggestion.
The current 0.0.7-SNAPSHOT version worked successfully on my host.
